runtime/interpreter/encode.go

```diff
@@ -1601,6 +1622,21 @@ var _ atree.TypeInfo = compositeTypeInfo{}
 
 const encodedCompositeTypeInfoLength = 3
 
+func (c compositeTypeInfo) IsComposite() bool {
+	return true
+}
+
+func (c compositeTypeInfo) Identifier() string {
+	// TODO: maybe improve this to be more efficient, etc.
+	return "composite(" + c.location.ID() + "/" + c.qualifiedIdentifier + ")"
+}
+
+func (c compositeTypeInfo) Copy() atree.TypeInfo {
+	// TODO: is creating a new compositeTypeInfo necessary?
+	// atree needs immutable TypeInfo or a copy of mutable TypeInfo because TypeInfo can be shared.
+	return NewCompositeTypeInfo(nil, c.location, c.qualifiedIdentifier, c.kind)
```

Comment:
Can this function just return `c` as-is, given it's a value type? The other `Copy` functions below do just that